From 29f717b3fc6c5a5dd5a042de02fe1f3a94472995 Mon Sep 17 00:00:00 2001
From: Radu-Andrei BULIE <radu.bulie@freescale.com>
Date: Wed, 31 Oct 2012 22:38:58 +0000
Subject: [PATCH 369/518] offload_use_case: Discard unreachable code

dbgfs_create_stats_file always returns value 0. Checking for
negative return value is useless.

Signed-off-by: Radu-Andrei BULIE <radu.bulie@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 .../staging/fsl_dpa_offload/usecases/dbgfs_ctl.c   |   16 ++--------------
 1 files changed, 2 insertions(+), 14 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/usecases/dbgfs_ctl.c b/drivers/staging/fsl_dpa_offload/usecases/dbgfs_ctl.c
index 13ae7cc..1e82789 100644
--- a/drivers/staging/fsl_dpa_offload/usecases/dbgfs_ctl.c
+++ b/drivers/staging/fsl_dpa_offload/usecases/dbgfs_ctl.c
@@ -106,7 +106,6 @@ int dbgfs_create_entry(int sa_id)
 {
 	char node_name[32];
 	struct dentry *dir;
-	int ret = 0;
 
 	struct dbgfs_sa_entry *node;
 
@@ -121,19 +120,8 @@ int dbgfs_create_entry(int sa_id)
 
 	node->dir = dir;
 
-	ret = dbgfs_create_stats_file("bytes", BYTES, sa_id, node);
-	if (ret < 0) {
-		pr_err("%s(%d) : Error creating 'bytes' entry (%d)\n",
-				__func__, __LINE__, ret);
-		return ret;
-	}
-
-	ret = dbgfs_create_stats_file("packets", PACKETS, sa_id, node);
-	if (ret < 0) {
-		pr_err("%s(%d) : Error creating 'packets' entry (%d)\n",
-				__func__, __LINE__, ret);
-		return ret;
-	}
+	dbgfs_create_stats_file("bytes", BYTES, sa_id, node);
+	dbgfs_create_stats_file("packets", PACKETS, sa_id, node);
 
 	list_add(&(node->list), &dbgfs_headlist);
 
-- 
1.7.5.4

